app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with model: @task, class: 'edit-task' do |f| %>
+
+  <p>Fill out the form to create/edit this task</p>
+
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <%= f.label :completion_date %>
+  <%= f.text_field :completion_date %>
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+
+
+  <%= f.submit "edit task", class: "task-form__submit-btn" %>
```

Comment:
I suggest passing a parameter into the partial, like we did in Ada-books, so the new and edit forms have different text on the submit button.